mbed_build/_internal/config/assemble_build_config.py

```diff
@@ -21,16 +22,22 @@ def assemble_config(mbed_target: str, mbed_program_directory: Path) -> Config:
     """
     target_source = Source.from_target(mbed_target, mbed_program_directory)
     mbed_lib_files = find_files("mbed_lib.json", mbed_program_directory)
-    return _assemble_config(target_source, mbed_lib_files)
+    mbed_program = MbedProgram.from_existing_local_program_directory(mbed_program_directory)
```

Comment:
I think these method names have changed in the latest release of mbed-project

Reply:
https://github.com/ARMmbed/mbed-build/pull/35/commits/98981df2f289be03578d919394e81a9e86abf4cf